pkg/adafruitio/adafruitio.go

```diff
@@ -109,6 +180,10 @@ func check(res *http.Response) error {
 	return nil
 }
 
+func (f Feed) String() string {
+	return fmt.Sprintf("%#v", f)
```

Comment:
This is not a good implementation of `String`. There isn't `Feed`-specific logic in here; a consumer that wanted to stringify a `Feed` this way could `fmt.Sprintf` themselves.